### PR TITLE
Add can sealer recipe.

### PR DESCRIPTION
**Pull request by Vasyan2006**
Add can sealer recipe, because it is impossible to find this essential tool.

### CODE REVIEW

**Comment by Zireael07** [comment]
I believe can sealer is a little bit too complex for a survivor to make...

**Comment by Vasyan2006** [comment]
Should we hide this recipe in lab journal Herrera, between microreactor CBM and teleport pad?

**Comment by pisskop** [comment]
You could start with not making it as complex as a quarterstaff.

**Comment by ZhilkinSerg** [comment]
See #20837.

**Comment by Night-Pryanik** [comment]
Oh, damn, sorry.

**Comment by smolbird** [comment]
Might recommend some degree of forging setup for the recipe. It's a manual can seamer, but I'm not sure how much complexity goes into those.

**Comment by Vasyan2006** [comment]
Edit recipe to add mechanics skill requirement, more tools and using welder intead of soldering iron.

This device can be easily crafted from `[ [ "assorted_mechanical_parts", 25 ] ]` with industrial turning, drilling and milling machinery (represented by toolbox). Forging equipment is not required. 
can-sealer-ives-way-front

**Comment by Vasyan2006** [comment]
This head can be replased with Γ-shape item. It's main purpose is to connect a crank with other parts. Casted protection over gears can be removed (it will grind only few fingers in the worst case). The gears and ball bearings in rollers are most complicated parts, but can be found in any neighbor garage.
Unlike weapons such mechanical household tools are not strictly limited by weight or ballansing. Most of the parts can be replaced with more heavy and uncomfortable scrap.

**Comment by kevingranade** [comment]
> This head can be replased with Γ-shape item.

Cool, find an example of someone doing that.

As far as I can tell, metal can sealing actually has extremely tight tolerances and requires very high pressures, which is why all the examples I can find have cast components, they aren't there to keep your fingers safe, they're there to hold the thing together.  These things seem to cost several hundred to several thousand dollars IRL, I suspect there's a good reason for that.

Casting the head is pretty doable, it's just some of the forging tools and a crucible and a heat source.

File: data/json/recipes/other/tool.json [hunk 1]
```diff
@@ -158,15 +158,19 @@
...
     "qualities": [
-      { "id": "SCREW", "level": 1, "amount": 1 },
-      { "id": "HAMMER", "level": 3, "amount": 1 }
+      { "id": "HAMMER", "level": 3 },
+      { "id": "SCREW", "level": 1 },
+      { "id": "WRENCH", "level": 2 },
+      { "id": "SAW_M", "level": 2 },
+      { "id": "GLARE", "level": 2 }
     ],
+    "tools": [ [ [ "welder", 10 ], [ "welder_crude", 15 ], [ "toolset", 15 ], [ "oxy_torch", 2 ] ] ],
```
Comment (Night-Pryanik) [inline data/json/recipes/other/tool.json]: You can replace all this tools with `welding_standard` requirement.

**Comment by Vasyan2006** [comment]
Sealers with price tag over $1000 have very hight durability (that is why they have casted parts) and will provide required sealing quality after 10000 cans. Food canned this way will last for many years.

But I want to add a tool slightly more complex compare to this one.

**Comment by kevingranade** [comment]
https://www.koerner-co.com/utensils-tools-accessories/103311-manual-round-can-sealer.html
"Manual can sealer only works on the round aluminium cans.
Closes the cans in seconds and with minimum effort, allows an hermetic and professional seal but is not suitable for keeping canned food during a long period of time."

In other words, it's a press-fit, not an actual seal.
Feel free to keep looking, or just add casting.

**Comment by Vasyan2006** [comment]
Add forge requirements and increase difficulty, as the requires metal casting now.

**Pull request merged**